Summary of Ristenpart 2009

nhwells opened the pull request:
Summary of Ristenpart 2009

Fixes #1

Commit:
Summary of Ristenpart 2009

Fixes #1

dmurvihill commented on bibliography.bib:
Editing will be way easier if you stick to 120-character lines and place each sentence on a new line.

dmurvihill commented on bibliography.bib:
```diff
@@ -7,6 +7,26 @@ @inproceedings{gullasch2011cache
+  year={2009},
+  publisher={ACM},
+  annote={
+    \setlength{\parskip}{1.5ex}
```
I'm not sure that this looks better. If you think it does, we should find a way of putting it in our preamble once instead of starting every annotation with it.

dmurvihill replied:
This line still bothers me, but it won't block the merge; I'll open an issue to find a better way of doing this.

dmurvihill commented on bibliography.bib:
Define Dom0 first.

dmurvihill commented on bibliography.bib:
If the paper cited potential attacks, go ahead and mention them here. One of the primary goals of this phase is to get an idea of the overall relationships between the related work.

In general, this annotation could use references to related work when appropriate.

dmurvihill commented on bibliography.bib:
Did the latter attack require knowledge of the target's creation parameters? Is this knowledge an attacker could reasonably expect to have?

dmurvihill commented on bibliography.bib:
Also, I don't fully understand the first method. Did it have one machine probe a bunch of other machines to see whether they were co-located? Which machines did the authors create?

Commit:
Ristenpart updates

fixed some formatting (hopefully?) and worked on addressing concerns
from the last version

dmurvihill commented on bibliography.bib:
I just noticed that this entry is never closed. BibTeX generates an error, but the system still manages to make the document.

dmurvihill commented on bibliography.bib:
Better, but also indent sentence continuations an extra notch. I know I'm being picky, but future Nate will thank you for the readability improvement.

dmurvihill commented on bibliography.bib:
You accidentally a word.

dmurvihill commented on bibliography.bib:
Still want to know how an attacker discovers these parameters in practice. Does the paper mention that problem at all?

dmurvihill commented:
Almost there; one content note, one code style note, one grammatical error, and one syntax error.

dmurvihill commented on bibliography.bib:
Also just noticed that these lines go out to 140 characters; 120 is a more common convention.

Vim/Emacs macros will work wonders here.

dmurvihill commented on bibliography.bib:
I just ran into an issue in the summary I am writing where the citation ended up widowed on the following line. I fixed it by using a non-breaking space (~) before the citation. I suggest you do the same.

dmurvihill commented:
I am ready to merge this as soon as this last round of comments is addressed.

Commit:
More Ristenpart changes
